### DISCUSSION TITLE
Using RG as a Custom Github Action

### DISCUSSION THREAD

**Discussion by h3ct0rjs** [BurntSushi/ripgrep#2740] [Q&A] [answered]
Hello Community, 

Hope you're doing great today, I have been checking the Github marketplace looking for a RipGrep Action, and since I was not able to find something I was wondering if is there any custom Github action or if not to build one. 

Thanks

**Comment by BurntSushi** [comment, marked as answer]
I'm not aware of one and I don't have any plans to build one or make one. I'm not even quite sure what it would do. Would it just install ripgrep? Chances are you can just install it with `apt-get install ripgrep` (or whatever). There are a variety of other ways to very easily install ripgrep, many of which are documented in the README. I don't really see any particular need for an entire Action just for that.

It might help to explain more of your use case.
